R_scripts/licorstat.R

```diff
@@ -28,7 +29,7 @@ dat <- filter(dat, Group == "Control")
 cat("Fitting full model...\n")
 library(nlme)
 mod_full <- nlme::lme(log(Flux) ~ T5 + T20 + SMoisture + I(SMoisture ^ 2), 
-                      random = ~ 1 | Dest_Salinity,
+                      random = ~ 1 | BAdat$BA_sqm,
```

Comment:
I think we want this to be a fixed, not random, effect.

Reply:
Yes this isn't the most recent version, `master` is most recent for licorstat.R. Things are a bit messy between all three branches currently. So i wanted to merge them and start again fresh

Reply:
You probably want to merge `master` into this first. We want to clean up any conflicts, etc., before merging _into_ master.